validation/validate.py

```diff
@@ -0,0 +1,267 @@
+import ast
+import asyncio
+import glob
+import json
+import os
+import time
+from datetime import datetime
+from typing import Dict, List
+
+import dotenv
+import pandas as pd
+from sqlalchemy.ext.asyncio import AsyncSession, create_async_engine
+from sqlalchemy.orm import sessionmaker
+
+from askametric.query_processor.query_processor import LLMQueryProcessor
+from askametric.validation.validation_processor import QueryEvaluator
+
+dotenv.load_dotenv()
+
+DATA_SOURCES_PATH = "data_sources"
```

Comment:
can we also allow this to be an input to the main function (added suggested code snippet for this below), instead of hard-coding it? O.w. we just end up having multiple copies of the same db files.